.github/workflows/CI.yml

```diff
@@ -0,0 +1,440 @@
+    strategy:
+      matrix:
+        python-version: [2.7, 3.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download our build
+        uses: actions/download-artifact@v1
+        with:
+          name: bdist-wheel-package
+          path: packages
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: python --version
+      - run: pip install --upgrade pip
+      - run: pip install --upgrade pytest
+      - name: Install our package
+        run: pip install $(find packages -name "*.whl")
+      - name: Assert is importable and show version
+        run: python -c "import sphinx_typo3_theme; print(sphinx_typo3_theme.__version__)"
+      - name: Assert is importable and show version_info
+        run: python -c "import sphinx_typo3_theme; print(sphinx_typo3_theme.version_info)"
+      - run: python tests/test_python_sphinx_typo3_theme.py
+      - name: make test TODO Make this work
+        if: 0
+        run: make test
+
+
+  passed-all-tests:
+    needs: [make-docs, make-test, sphinx-quickstart]
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo passed all tests
+
+
+  upload-to-cdn:
+    needs: [passed-all-tests]
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo 'github.event_name  :' ${{ github.event_name }}
+      - run: echo 'github.event.action:' ${{ github.event.action }}
+      - name: Download our package
+        uses: actions/download-artifact@v1
+        with:
+          name: bdist-wheel-package
+          path: packages
+      - name: Find wheel and version
+        id: get-version
+        run: |
+          the_wheel=$(find packages -name "*.whl")
+          echo "the wheel package is: '"${the_wheel}"'"
+          unzip ${the_wheel}
+          find .
+          echo ::set-output name=version1::$(echo ${the_wheel} | sed -e 's/^.*sphinx_typo3_theme-\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\).*/\1/')
+          echo ::set-output name=version2::$(echo ${the_wheel} | sed -e 's/^.*sphinx_typo3_theme-\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\).*/\1.\2/')
+          echo ::set-output name=version3::$(echo ${the_wheel} | sed -e 's/^.*sphinx_typo3_theme-\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\).*/\1.\2.\3/')
+          env
+      - name: Show version1, version2, version3
+        run: |
+          echo "version1: ${{ steps.get-version.outputs.version1 }}"
+          echo "version2: ${{ steps.get-version.outputs.version2 }}"
+          echo "version3: ${{ steps.get-version.outputs.version3 }}"
+      - name: Remove some files from upload
+        run: |
+           find sphinx_typo3_theme/static -name __init__.py -delete
+           find sphinx_typo3_theme/static -name *_t -delete
+      - name: Upload static as artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: assets-for-cdn
+          path: sphinx_typo3_theme/static
+      - name: TODO Upload to Azure.
```

Comment:
we should only deploy on released versions with the full version number.
in previous builds I've also deployed the master branch that was more for internal testing purposes than external usage.

1. it avoids http caching issues
2. we would need to invalidate the caches on the cdn that makes additional work
3. we should not change the appearance of a rendered documentation.
4. this build should run, after we published to pypi
5. the folder that will be uploaded must contain the following structure `theme/sphinx_typo3_theme/<version/branch>` the cdn deployment will simply upload this structure

Reply:
Ok. Will do. Although I'm regretting that we can't do apply instant changes, for example at patch level, to all manuals.